README.md

```diff
@@ -7,7 +7,84 @@
 - Numbers = `[0-9]* | [0-9]+.[0-9]*`
 - LPAR = `(`, RPAR = `)`
 - Separator = `; | ,`
+- String_literal = ` " `
```

Comment:
Is this regular expression? I feel like we should add that we accept any kind of characters inside the pair of double quotes.